staging/src/k8s.io/apimachinery/pkg/util/runtime/runtime.go

```diff
@@ -101,6 +111,26 @@ func logError(err error) {
 	glog.ErrorDepth(2, err)
 }
 
+type rudimentaryErrorBackoff struct {
+	// TODO(lavalamp): use the clock for testability. Need to move that
+	// package for that to be accessible here.
+	lastErrorTime time.Time
+	minPeriod     time.Duration
+	lock          sync.Mutex
```

Comment:
nit, lock should be the field above the thing it locks and should be named lasteErrorTimeLock or something.